docs/content/preview/develop/best-practices-ysql.md

```diff
@@ -39,6 +39,15 @@ Use [covering indexes](../../explore/indexes-constraints/covering-index-ysql/) t
 For more details, see [Avoid trips to the table with covering indexes](https://www.yugabyte.com/blog/multi-region-database-deployment-best-practices/#avoid-trips-to-the-table-with-covering-indexes).
 {{</tip>}}
 
+## Faster writes with partial indexes
```

Comment:
Suggestion: Speed up your queries with ... 

As it affects both reads and writes.

Reply:
Reads will not be faster though? Just that the index will need less maintenance. But a query using it should have the same performance.